### ISSUE TITLE
Map from DLC Vive La France & Etc

### ISSUE THREAD

**Issue by Rendername** [mike-koch/ets2-mobile-route-advisor#131]
Hello. Tell me please, any plans to add maps of the DLC Vive La France? Maybe there's some manual for independent attempts to "make"?

**Comment by Koenvh1** [comment]
There certainly are plans... It shouldn't take too long, but I need to find the time to do it, and I just haven't come around to doing it yet. As for a manual; you may want to look at this: https://github.com/Koenvh1/ETS2-City-Coordinate-Retriever/blob/master/Capturing%20maps.md
However, the mod files to make the map "capture-ready" may be broken (not quite sure). Give it a shot if you want 😉

**Comment by Rendername** [comment]
If need someone be, here is a map with DLC, but it is not open all POI and the color of the road - grey. Can someone come in handy. Pull Request did not do, because probably it doesn't make sense. 💯%
[0map.zip](https://github.com/mike-koch/ets2-mobile-route-advisor/files/908242/0map.zip)

**Comment by mike-koch** [comment]
@Rendername: I saw this notification come into my email, but I don't see the message here anymore. Any reason why you deleted the message? Perhaps it may have been useful :P

> If need someone be, here is a map with DLC, but it is not open all POI and the color of the road - grey. Can someone come in handy. Pull Request did not do, because probably it doesn't make sense. :100:%
0map.zip

**Comment by Rendername** [comment]
> @Rendername: I saw this notification come into my email, but I don't see the message here anymore. Any reason why you deleted the message? Perhaps it may have been useful 

Ah. I checked, in fact, a lot of mismatches for the coordinates. So while deleted the post. I'll try again in a manual fix it and publish an archive of the tiles.

**Comment by Rendername** [comment]
Fixed. 
In this map there DLC Vive La France. How to used? It is sufficient to replace the folder:
> \ets2-mobile-route-advisor\maps\ets2\tiles
[0map.zip](https://github.com/mike-koch/ets2-mobile-route-advisor/files/922476/0map.zip)

**Comment by mike-koch** [comment]
I'll try to take a look at the map later tonight to see how it looks :smiley_cat:

**Comment by Rendername** [comment]
It looks not perfect, but even so. If someone can do it better - of course that would be great. 🤔

**Comment by Rendername** [comment]
Anyone tested?

![Image](https://www.tuxboard.com/photos/2015/12/confused-travolta-code.gif)

**Comment by mike-koch** [comment]
@Rendername  So I finally got around to testing your map that I was going to look at a month ago (sorry for the delay, I upgraded HDDs and had to reinstall everything and just didn't feel like setting up the telemetry stuff for a while ☹️ ).  The map's sufficient for the next release from what I noticed (a quick run from London to Liverpool... going to test some stuff in France soon).  

I do have one request though for you. Could you re-capture the map, but remove all point of interest icons except for the company names and fuel icons? I noticed that I couldn't see a roundabout because it was covered in "?" icons.  In previous versions, only the company names/icons and fuel points were displayed.

**Comment by mike-koch** [comment]
@Rendername did you get a chance to read my previous comment?

**Comment by augustocb23** [comment]
Hi,
Do you have any idea about when will be the map released?
Will be it "auto-updated" or I need to come here to download? If need manual update, how can I subscribe for News about it?

Thx!

**Comment by mike-koch** [comment]
It'll be released when it's ready :wink: I'm hoping the updated map from Rendername will be done soon so the better version can be included, so I'm basically waiting on just that.

The best place to check for updates is at http://forum.scssoft.com/viewtopic.php?f=34&t=178742. In addition, the "about" tab will display an update message if there is a newer version available for download.

**Comment by Rendername** [comment]
@mike-koch 
Hello again. I tried to re-capture the map a few days ago, taking a profile with an open 100% map where icons are missing "?". But with 4 attempts and failed to obtain a normal image. This is due to the fact that the script is somehow not properly glues to the screenshots. Unfortunately now moved out of the house for a month, but I'll try to do it on the laptop. The main problem is offset screenshot images, for this reason the final variant is broken. :(

**Comment by Rendername** [comment]
Updated archive.
Here removed "?" icons in this map. 😛
Archive in the attachment. 
Mirror: https://yadi.sk/d/PnTbDObF3K6Xfv
Server github:
[ets2-map-all-dlc.zip](https://github.com/mike-koch/ets2-mobile-route-advisor/files/1072836/ets2-map-all-dlc.zip)

**Comment by mike-koch** [comment]
Looks good :+1: I'll try to bundle a release soon :smile_cat:

**Comment by mike-koch** [comment]
Merged into `next-release`. Closing.
